src/rust/lib/graph-editor/src/lib.rs

```diff
@@ -324,6 +324,8 @@ ensogl::def_command_api! { Commands
     cycle_visualization_for_selected_node,
     /// Enter the last selected node.
     enter_selected_node,
+    /// Steps out of the current node, popping the topmost stack frame from the crumb list.
+    step_out_of_node,
```

Comment:
if we have "enter_..._node", this should be "exit_current_node" instead. Or we should use "step_in ...", but I prefer enter/exit.

Reply:
unified to use enter/exit words